connector/mqtt/vernemq/v2k-brigde/bin/scripts_tls/retrieveCRL.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+#########################################################
+#########################################################
+
+
+BASE_DIR=${BASE_DIR:-"/v2k_bridge"}
+
+. "${BASE_DIR}"/bin/scripts_tls/_initVariables.sh
+
+echo "Retrieve crl from  ${certCAName} CA : ${certEjbcaApiUrl}/ca/${certCAName}/crl?update=true"
+certCrl=$(curl --silent -X GET "${certEjbcaApiUrl}"/ca/"${certCAName}"/crl?update=true \
+-H "Content-Type:application/json" \
+-H "Accept:application/json" | jq '.CRL' -r)
+
+(echo  "-----BEGIN X509 CRL-----"
+echo "${certCrl}"
+echo "-----END X509 CRL-----"  ) > "${certDir}"/tempcrl.crl
```

Comment:
This command should produce the same result, with the advantage of using only one `echo` command.
```
echo  "-----BEGIN X509 CRL-----
${certCrl}
-----END X509 CRL-----" > "${certDir}/tempcrl.crl"
```
In this case, the `echo` command ends only when it closes quotation marks.

This same tip goes for other scripts!